src/sensitivities/linalg/generic.jl

```diff
@@ -134,3 +134,10 @@ function ∇(B̄::A, ::typeof(kron), ::Type{Arg{2}}, p, Y::A, Ȳ::A, A::A, B::A
     end
     return B̄
 end
+
+import Base: +
+@explicit_intercepts Base.:+ Tuple{A, UniformScaling}
```

Comment:
You shouldn't need the explicit `Base.:+` qualification here since you're `import`ing `+` above.